### PR TITLE
Fix import csv rows

### PR DESCRIPTION
**Pull request by Danble**
#### Checklist before marking ready to merge
Please keep it in draft mode until these are completed:
- [x] Equal time was spent cleaning the code as writing it (Boy Scout Rule)
  - [x] Functions
    - [x] Functions that don't belong in Svelte components are extracted out into `.ts` files
    - [x] Functions are short and well named
    - [x] Concise tests are written for all functions
  - [ ] Classes (a Svelte Component is a Class)
    - [ ] Svelte components are broken down into smaller components so that each component is responsible for one thing (Single Responsibility Principle)
    - [ ] Stories/variants are written to describe use cases
  - [ ] Comments are only included when absolutely necessary information that cannot be explained in code is needed

### CODE REVIEW

File: packages/scripts/import/parse-csv.ts [hunk 1]
```diff
@@ -1,14 +1,29 @@
 import { readFileSync } from 'fs';
 import { parse } from 'csv-parse/sync';
 
-export function parseCSVFrom(file: string): Record<string, any>[] {
-  return parse(file, {
+export function parseCSVFrom(contents: string): Record<string, any>[] {
+  return parse(contents, {
     columns: true,
     skip_empty_lines: true,
+    trim: true,
```
Comment (jacob-8) [inline packages/scripts/import/parse-csv.ts:8]: @Danble this is all that's needed. https://csv.js.org/parse/options/trim/

It's easy because the tool already provides that option, but if it hadn't provided it, new code should be placed into parseCSVFrom and not convertJsonRowToEntryFormat since trimming whitespace is part of parsing and not converting.

Also, keep your tests simpler, note how an extremely small test can check whitespace in mutliple header and content row contexts.

**Review by jacob-8 — APPROVED** [review]
